src/qml/controls/ExternalLink.qml

```diff
@@ -45,9 +45,7 @@ AbstractButton {
             Layout.fillWidth: true
             active: root.description.length > 0
             visible: active
-            sourceComponent: Text {
-                font.family: "Inter"
-                font.styleName: "Regular"
+            sourceComponent: CoreText {
```

Comment:
```diff

diff --git a/src/qml/controls/CoreText.qml b/src/qml/controls/CoreText.qml
index 1029cd0af..56e122a9a 100644
--- a/src/qml/controls/CoreText.qml
+++ b/src/qml/controls/CoreText.qml
@@ -7,11 +7,12 @@ import QtQuick.Controls 2.15
 
 Text {
     property bool bold: false
+    property bool wrap: true
     color: Theme.color.white
     font.family: "Inter"
     font.styleName: bold ? "Semi Bold" : "Regular"
     font.pixelSize: 13
     horizontalAlignment: Text.AlignHCenter
     verticalAlignment: Text.AlignVCenter
-    wrapMode: Text.WordWrap
+    wrapMode: root.wrap ? Text.WordWrap : Text.NoWrap
 }
diff --git a/src/qml/controls/ExternalLink.qml b/src/qml/controls/ExternalLink.qml
index ab10d1b89..2e189037a 100644
--- a/src/qml/controls/ExternalLink.qml
+++ b/src/qml/controls/ExternalLink.qml
@@ -48,6 +48,7 @@ AbstractButton {
             sourceComponent: CoreText {
                 font.pixelSize: root.descriptionSize
                 color: root.textColor
+                wrap: false
                 textFormat: Text.RichText
                 text: root.description
```

Reply:
Something like the above would fix the [wrapping issue](https://github.com/bitcoin-core/gui-qml/pull/268#pullrequestreview-1308709522) I noticed on macOS